app/lib/Plugins/SearchEngine/Elastic8/FieldTypes/DateRange.php

```diff
@@ -154,4 +154,8 @@ function getFiltersForTerm($term, ?string $field = null): array {
 		return $return;
 	}
 
+	public function getDefaultSuffix(): string {
```

Comment:
@kehh this is the line causing the error while indexing. We can remove this, or update parseElasticsearchDateRange() to hardcode the text suffix for the string representation of the date (or remove, unsure if it is needed)

```
public function parseElasticsearchDateRange($content, $historic = false): array {
		$return = [];
		$return[$this->getDataTypeSuffix(FieldType::SUFFIX_TEXT)] = $content;
		...
	}
		```

Reply:
Haha sorry you went down that rabbit hole too ;)